lib/tools/apk-signing.js

```diff
@@ -284,33 +292,40 @@ apkSigningMethods.checkApkCert = async function checkApkCert (appPath, pkg) {
     appPath = await this.extractBaseApk(appPath);
   }
 
+  const {
+    requireDefaultCert = true,
+  } = opts;
   try {
     await getApksignerForOs(this);
     const output = await this.executeApksigner(['verify', '--print-certs', appPath]);
-    if (!_.includes(output, DEFAULT_CERT_DIGEST)) {
-      log.debug(`'${appPath}' is signed with non-default certificate`);
-      return false;
+    if (_.includes(output, DEFAULT_CERT_DIGEST)) {
+      log.debug(`'${appPath}' is signed with the default certificate`);
+    } else {
+      log.debug(`'${appPath}' is signed with a non-default certificate`);
     }
-    log.debug(`'${appPath}' is already signed.`);
-    return true;
+    return !requireDefaultCert || (requireDefaultCert && _.includes(output, DEFAULT_CERT_DIGEST));
```

Comment:
I don't really understand what the second instance of `requireDefaultCert` is doing here. It will always be `true` in this case, no?

Reply:
true, updated